lib/sidekiq/displayers/active_job.rb

```diff
@@ -0,0 +1,77 @@
+module Sidekiq
+  module Displayers
+    module ActiveJob
+      ACTIVE_JOB_PREFIX = "_aj_"
+      GLOBALID_KEY = "_aj_globalid"
+      SYMBOL_KEYS_KEY = "_aj_symbol_keys"
+      RUBY2_KEYWORDS_KEY = "_aj_ruby2_keywords"
+
+      def self.valid_for?(job)
```

Comment:
I'd like to see an effort to move away from global/class methods. Store an instance in S::Config and reference that instance rather than global methods and ifs everywhere.

Reply:
@mperham Honestly, what's the difference? I can make it into a singleton but isn't it functionally identical?